.github/actions/rust/action.yaml

```
@@ -45,7 +45,7 @@ runs:
       with:
         repo-token: ${{ inputs.github-token }}
     - name: Use sccache
-      uses: mozilla-actions/sccache-action@v0.0.7
+      uses: mozilla-actions/sccache-action@v0.0.8
```

```suggestion
      uses: mozilla-actions/sccache-action@v0.0.9
```
(fixes https://github.com/Mozilla-Actions/sccache-action/issues/191 )

---

thanks--didn't see this since it wasn't marked as a release